.github/workflows/frontend.yml

```diff
@@ -40,3 +40,9 @@ jobs:
         run: |
           sudo sysctl fs.inotify.max_user_watches=524288
           node scripts/cli-e2e-test.js
+      - name: yarn lint, test after plugin creation
+        run: |
+          yarn lint
+          yarn test
```

Comment:
This'll test and lint the entire project, which isn't really what we want, that's taken care of by in a separate task. I think we can solve it by telling GitHub workflows to run the commands inside the plugin folder only.

Reply:
Ok, I thought it needed to consider the new plugin created, not just them (plugins folders). I can continue on this and make this change, no problem.

Reply:
Yeah only running tests inside `plugins/test-plugin` was the idea